app/code/Vladimirl/Chatter/Model/ChatMessage.php

```diff
@@ -0,0 +1,19 @@
+<?php
+declare(strict_types=1);
+
+namespace Vladimirl\Chatter\Model;
+
+use Magento\Framework\Model\AbstractModel;
+use Vladimirl\Chatter\Model\ResourceModel\ChatMessage as ResourceModel;
+
+class ChatMessage extends AbstractModel
```

Comment:
@Vladimir-L Use full class names in `extends`, `implements`, in constructor and in the property definition.